ui/tests/helpers/a11y-audit.js

```diff
@@ -1,16 +1,25 @@
 import a11yAudit from 'ember-a11y-testing/test-support/audit';
 
-export default async function defaultA11yAudit(...overriddenRules) {
+function appendRuleOverrides(overriddenRules) {
   const rules = {
     'color-contrast': {
-      enabled: false
+      enabled: false,
     },
     'heading-order': {
-      enabled: false
-    }
+      enabled: false,
+    },
   };
 
-  overriddenRules.forEach(rule => rules[rule] = { enabled: false });
+  overriddenRules.forEach(rule => (rules[rule] = { enabled: false }));
+
+  return rules;
+}
+
+export default async function defaultA11yAudit(...overriddenRules) {
+  await a11yAudit({ rules: appendRuleOverrides(overriddenRules) });
+}
 
-  await a11yAudit({rules});
+export async function componentA11yAudit(element, assert, ...overriddenRules) {
+  await a11yAudit(element, { rules: appendRuleOverrides(overriddenRules) });
+  assert.ok(true, 'a11y audit passes');
```

Comment:
Sticking the assertion in the helper is a neat workaround for the littering of `assert.ok(true)`s the acceptance test a11y audits suffered from.

Does it make sense to update `defaultA11yAudit` accordingly?

Reply:
Good idea, thanks, I added 227abdd to do that!